components/Starknet/modules/architecture-and-concepts/pages/network-architecture/transaction-life-cycle.adoc

```diff
@@ -49,11 +49,16 @@ The transaction has no execution status.
 
 [NOTE]
 ====
-A `REJECTED` transaction is stored in the mempool. You cannot send another transaction with the same transaction hash.
+A rejected transaction is stored in the mempool. You cannot send another transaction with the same transaction hash.
 ====
 |`ACCEPTED_ON_L2` |The transaction was executed and entered an actual created block on L2.
 |`ACCEPTED_ON_L1` |The transaction was accepted on Ethereum.
-.2+|*Execution* |`REVERTED` |The transaction passed validation but failed during execution in the sequencer. It is included in the block with the status `REVERTED`.
+.2+|*Execution* |`REVERTED` a|The transaction passed validation but failed during execution in the sequencer. It is included in the block with the status `REVERTED`.
+
+[NOTE]
+====
+Since only `INVOKE` transactions have an execution phase, `DEPLOY_ACCOUNT` and `DECLARE` transactions cannot be reverted. If the `+__VALIDATE_DEPLOY__+` or `+__VALIDATE_DECLARE__+` function, respectively, fail when run in the sequencer, then the transaction will be rejected.
```

Comment:
```suggestion
Since only `INVOKE` transactions have an execution phase, `DEPLOY_ACCOUNT` and `DECLARE` transactions cannot be reverted. If either the `+__VALIDATE_DEPLOY__+` or the `+__VALIDATE_DECLARE__+` function fails when run in the sequencer, then the transaction is rejected.
```